pyomo/solvers/tests/checks/test_gurobi.py

```diff
@@ -0,0 +1,43 @@
+import pyomo.common.unittest as unittest
+from unittest.mock import patch, MagicMock
+
+try:
+    from pyomo.solvers.plugins.solvers.GUROBI_RUN import gurobi_run
+    from gurobipy import GRB
+    gurobipy_available = True
+except:
+    gurobipy_available = False
+
+class GurobiTest(unittest.TestCase):
+
+    @unittest.skipIf(not gurobipy_available,
```

Comment:
Suggestion: move this first `skipIf` to right before the class declaration. That should fix the failure on the `slim` build.